clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -2302,6 +2302,7 @@ private void close(long timeoutMs, boolean swallowException) {
         Utils.closeQuietly(fetcher, "fetcher", firstException);
         Utils.closeQuietly(interceptors, "consumer interceptors", firstException);
         Utils.closeQuietly(metrics, "consumer metrics", firstException);
+        Utils.closeQuietly(kafkaConsumerMetrics, "kafka consumer metrics", firstException);
```

Comment:
Hey Colin, kafkaConsumerMetrics is not a Closeable type. Can you confirm this compiles?